src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -589,31 +602,34 @@ private boolean retryUnsubscribe(UnsubAckPacket.UnsubAckReasonCode rc) {
 
     @Override
     public void start()  {
-        if (clientStarted) {
-            return;
+        if (isSSL()) {
```

Comment:
is `start` going to be called again if the ssl setting changes?

Reply:
right now if ssl settings change, a new LocalMqtt5Client instance will be created.  after this change, we can use reset() instead of reinstalling for those config changes, and we'd need to revisit this.  but  this change addresses an issue where we were unsubscribing and resubscribing to keystore updates during a reset